src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsnonassociationsapi/config/HmppsNonAssociationsApiExceptionHandler.kt

```diff
@@ -132,13 +132,14 @@ class HmppsNonAssociationsApiExceptionHandler {
   @ExceptionHandler(ResponseStatusException::class)
   fun handleResponseStatusException(e: ResponseStatusException): ResponseEntity<ErrorResponse?>? {
     log.debug("Response status exception caught: {}", e.message)
+    val reason = e.reason ?: "Unknown error"
```

Comment:
might as well send "Unknown error" rather than null so that `ErrorResponse`’s fields do not need to be nullable